ql/src/java/org/apache/hadoop/hive/ql/ddl/DDLOperationContext.java

```diff
@@ -34,19 +34,19 @@
 public class DDLOperationContext {
   private final Hive db;
   private final HiveConf conf;
-  private final DriverContext driverContext;
+  private final DriverQueue driverQueue;
```

Comment:
not sure about this; I think we might want to add this "queue" to the driverContext - as we have a lot of other stuff hanging around here...which could be moved into the context (conf?)

I would never think about asking for informations from a "queue"...(beyond the elements it might contain)

Reply:
fixed.